doc/modules/cross_validation.rst

```diff
@@ -172,6 +171,54 @@ validation iterator instead, for instance::
 
     See :ref:`combining_estimators`.
 
+
+.. _multimetric_cross_validation:
+
+Multiple metric evaluation and change in return type
+----------------------------------------------------
+Starting from version 0.19, it is possible to specify multiple metrics for
```

Comment:
This shouldn't be in here. The docs are always for the newest version. That's for whatsnew.